src/sidebar/test/oauth-auth-test.js

```diff
@@ -53,6 +53,18 @@ describe('oauth auth', function () {
       });
     });
 
+    it('should raise if a grant token request fails', function () {
```

Comment:
> I suppose your are requesting an access token as in that's the thing that you're asking for in response.

Yes, exactly.

> But that is also true of refresh token requests, so would you call those access token requests as well?

Yes, but the test might be worded as "access token refresh" rather than "access token request" to differentiate the two.

The other option would be to describe this test as a _grant token exchange_, since that is what the client provides in return for the access token.